### PR TITLE
remove defer

### PR DESCRIPTION
**Pull request by jnurthen**
resolves ID issues (currently being tracked in https://github.com/w3c/aria-common/issues/7 )

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/core-aam/pull/91.html" title="Last updated on Mar 25, 2021, 8:53 PM UTC (8c3538e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/core-aam/91/56e9175...8c3538e.html" title="Last updated on Mar 25, 2021, 8:53 PM UTC (8c3538e)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -138,16 +138,18 @@
...
     
     // Working group info
-    wg:           "Accessible Rich Internet Applications Working Group",
-    wgURI:        "https://www.w3.org/WAI/ARIA/",
-    wgPublicList: "public-aria",
+	group: "aria",
+
+    // wg:           "Accessible Rich Internet Applications Working Group",
+    // wgURI:        "https://www.w3.org/WAI/ARIA/",
+    // wgPublicList: "public-aria",
```
Comment (joanmarie) [inline index.html:145]: Shouldn't we just delete these rather than comment them out?

File: index.html [hunk 2]
```diff
@@ -138,16 +138,18 @@
     */
     
     // Working group info
-    wg:           "Accessible Rich Internet Applications Working Group",
-    wgURI:        "https://www.w3.org/WAI/ARIA/",
-    wgPublicList: "public-aria",
+	group: "aria",
```
Comment (joanmarie) [inline index.html:141]: Nit: whitespace differences.

**Review by joanmarie — APPROVED** [review]
LGTM, but please see the inline comments.

File: index.html (hunk 2 above)
Comment (jnurthen) [reply index.html:141]: not sure there is any consistency here in this part of the file on whitespace

File: index.html (hunk 1 above)
Comment (jnurthen) [reply index.html:145]: +1

**Pull request merged**